azure-pipelines.yml

```diff
@@ -0,0 +1,72 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - "*"
+
+pr: none
+
+pool:
+  vmImage: 'Ubuntu-16.04'
+
+variables:
+  imageName: 'manage-courses-support-poc'
+  # Support multiple docker-compose files to allow travis and az pipelines to co exist: https://docs.docker.com/compose/extends/#multiple-compose-files
+  dockerOverride: 'docker-compose -f docker-compose.yaml -f docker-compose.az.yml'
```

Comment:
Just curious ... below I see `$DOCKER_OVERRIDE` being used, which I would have to guess is set by this setting above, but then there's also `$(imageName)` used, which I would've guessed could've been `$IMAGE_NAME`, based on the setting in this section. Is that the case? Or has `DOCKER_OVERRIDE` been set by something else?

Reply:
Good point and I will test this today to see if we can clean up the build a tiny bit